lib/modules/styleTweaks.js

```diff
@@ -330,6 +331,7 @@ async function toggleSubredditStyle(toggle, subreddit = currentSubreddit()) {
 
 		BodyClasses.toggle(toggle, 'res-srstyle-enabled');
 		BodyClasses.toggle(!toggle, 'res-srstyle-disabled');
+		BodyClasses.toggle(!toggle, 'no-subreddit-style');
```

Comment:
It looks like `res-srstyle-disabled` is already used for this, so you can remove this line and change the other line to add `res-srstyle-disabled`

Reply:
For some reason I assumed the issue was asking for an additional class on top of `res-srstyle-disabled` (which wouldn't really make sense looking back on it).  I'll change it to just use the existing "no subreddit style" class.